src/index.html

```diff
@@ -8,6 +9,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <!-- <h1>Stars</h1> -->
+    <div class="div">
+      <ul class="stars stars--0">
+        <li class="stars__item stars__item--1"></li>
+        <li class="stars__item stars__item--1"></li>
+        <li class="stars__item stars__item--1"></li>
+        <li class="stars__item stars__item--1"></li>
+        <li class="stars__item stars__item--1"></li>
```

Comment:
you should implement stars only with css without extra classes
```suggestion
        <li class="stars__item"></li>
        <li class="stars__item"></li>
        <li class="stars__item"></li>
        <li class="stars__item"></li>
        <li class="stars__item"></li>
```